pkg/controller/localvolume/localvolume_controller.go

```diff
@@ -37,25 +33,12 @@ import (
 type localDiskData map[string]map[string]string
 
 const (
-	// Name of the component
-	componentName = "local-storage-operator"
-
-	localDiskLocation            = "/mnt/local-storage"
-	provisionerServiceAccount    = "local-storage-admin"
-	provisionerPVRoleBindingName = "local-storage-provisioner-pv-binding"
-	provisionerNodeRoleName      = "local-storage-provisioner-node-clusterrole"
-
-	localVolumeRoleName        = "local-storage-provisioner-cr-role"
-	localVolumeRoleBindingName = "local-storage-provisioner-cr-rolebinding"
-
-	defaultPVClusterRole           = "system:persistent-volume-provisioner"
-	provisionerNodeRoleBindingName = "local-storage-provisioner-node-binding"
-	ownerNamespaceLabel            = "local.storage.openshift.io/owner-namespace"
-	ownerNameLabel                 = "local.storage.openshift.io/owner-name"
+	componentName       = "local-storage-operator"
+	localDiskLocation   = "/mnt/local-storage"
+	ownerNamespaceLabel = "local.storage.openshift.io/owner-namespace"
+	ownerNameLabel      = "local.storage.openshift.io/owner-name"
 
```

Comment:
These new names of owners don't work with existing logic of preventing `localvolume` object deletion and hence will result in a regression.

Reply:
checking

Reply:
Adding these labels back  for localvolume because removing labels doesn't sound like a good idea

Reply:
Done